esm-samples/jsapi-deno/request.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+deno run --allow-net --allow-env --allow-read request.ts
```

Comment:
Minor - need an extra line return at the end.

Reply:
done